incubator/elasticsearch/templates/master-statefulset.yaml

```diff
@@ -127,3 +127,23 @@ spec:
       resources:
         requests:
           storage: "{{ .Values.master.storage }}"
+---
+{{- if .Values.master.budget.minAvailable -}}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: "{{ template "elasticsearch.master.fullname" . }}-master-budget"
```

Comment:
Should not need the ending suffix, assuming there is no specific reason behind this choice.